src/plugins/data/public/search/legacy/default_search_strategy.ts

```diff
@@ -30,34 +31,47 @@ export const defaultSearchStrategy: SearchStrategyProvider = {
   },
 };
 
-function msearch({
-  searchRequests,
-  legacySearchService,
-  config,
-  esShardTimeout,
-}: SearchStrategySearchParams) {
-  const es = legacySearchService.esClient;
-  const inlineRequests = searchRequests.map(({ index, body, search_type: searchType }) => {
-    const inlineHeader = {
-      index: index.title || index,
-      search_type: searchType,
```

Comment:
What was the `searchType` parameter used for?

Reply:
Is this related?
https://www.elastic.co/guide/en/elasticsearch/reference/7.8/search-request-search-type.html

Reply:
Correct, I looked into it and couldn't find a case where `searchType` was actually set -- it is always `undefined`.